src/main/java/org/fcrepo/repository/FedoraResourceImport.java

```diff
@@ -45,21 +46,44 @@
      */
     public static void main(final String[] args) {
 
-        final String fcrepoUrl = System.getProperty("fcrepo.url", "http://localhost:8080/rest/");
-        final String resourcesDir = System.getProperty("resources.dir", ".");
+        String fcrepoUrl = System.getProperty("fcrepo.url", "http://localhost:8080/fcrepo/rest/");
+        if (!fcrepoUrl.endsWith("/")) {
+            fcrepoUrl = fcrepoUrl + "/";
+        }
+        final String resourcesDir = System.getProperty("resources.dir", "./src/main/resources/data");
```

Comment:
All of the resources for webac are in the following directory:
`src/main/resources/data/acls` no?
If so, it seems like `resourcesDir` should reflect that fact.